cms/taxonomy/models.py

```diff
@@ -0,0 +1,114 @@
+import typing
+from typing import Any, ClassVar, Optional
+
+from django.db import models
+from django.db.models import QuerySet, UniqueConstraint
+from modelcluster.fields import ParentalKey
+from treebeard.mp_tree import MP_Node
+from wagtail.admin.panels import FieldPanel
+from wagtail.search import index
+
+BASE_TOPIC_DEPTH = 2
+
+
+class TopicManager(models.Manager):
+    def get_queryset(self) -> QuerySet:
+        """Filter out the root topic from all querysets."""
+        return super().get_queryset().filter(depth__gt=1)
+
+    def root_topic(self) -> "Topic":
+        return typing.cast(Topic, super().get_queryset().filter(depth=1).first())
```

Comment:
Worth adding a comment to say, root will always exist as it is created via the data migration because mypy complaining it can be None is valid.

Reply:
Addressed in https://github.com/ONSdigital/dis-wagtail/pull/104/commits/89fb9c9c8c63153bba489e9abedc3084ec285310